### ISSUE TITLE
performance changed in latest version?

### ISSUE THREAD

**Issue by david-jankoski** [hannesmuehleisen/MonetDBLite#185]
Hey Hannes & Team,

This is not a "nice" issue report but i was just wondering if you noticed as well performance degradation with the latest version of MonetDBLite?  
After installing the latest (0.4.1) queries that were fast somehow now take much longer to run.  
I will revert back to an older version and do some timings to support the claim, but was just curious if others had similar experience.

thanks for all your hard work. 
david

**Comment by hannesmuehleisen** [comment]
Any reproducible example you could make?

**Comment by hannesmuehleisen** [comment]
still an issue in dev version?

**Comment by david-jankoski** [comment]
well...it's kinda tricky mostly because of the data used in the process - but i will try and come up with one. until then i will close this issue and reopen if it makes sense...  
here's roughly what i am witnessing though:  

- a have a biggish .csv (14 GB), read it in with `monet.read.csv()`
- run a script that filters some subset of data and runs some models

now - since i updated to 0.4.1 there is a noticable slowdown at one part of the script (nothing else was changed in the script).  Going back to  0.3.2 with  
```
devtools::install_github(
  "hannesmuehleisen/MonetDBLite", 
  ref = "fcbb2068fe839941cbe0b3eb35108a2f3a8139a5"
  )
```  
seems to go back to "normal" and script finishes in "noticably" shorter time. I'm using all these vague words but i hope to provide you with some workable example.  
In each case i re-generate the monetdblite instance with the `monet.read.csv` from scratch with the same .csv file and run my script.  
I checked also with your latest commit and situation is the same.
